payments-core/src/main/java/com/stripe/android/paymentsheet/BaseAddPaymentMethodFragment.kt

```diff
@@ -148,6 +149,11 @@ internal abstract class BaseAddPaymentMethodFragment(
             ComposeFormDataCollectionFragment.EXTRA_PAYMENT_METHOD,
             paymentMethod.name
         )
+        addSaveForFutureUseArguments(
```

Comment:
@brnunes-stripe Here is where it is adding the save for future usage state in the fragment.